metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -110,20 +147,38 @@ def __init__(self, config, ctx):
 
     def _compute_big_query_lineage(self) -> None:
```

Comment:
After the lineage metadata is computed, could we just put a log statement say how many log entries were there in the dictionary. This will help us debug in case if there are some issues in querying the logs.

Reply:
Done!